app/models/manageiq/providers/vmware/infra_manager/inventory/persister.rb

```diff
@@ -44,6 +48,11 @@ def initialize_inventory_collections
     add_collection(infra, :orchestration_templates)
   end
 
+  def initialize_tag_mapper
+    @tag_mapper = ContainerLabelTagMapping.mapper
```

Comment:
Seems this class name should be changed eventually

Reply:
Yes definitely

Reply:
to `TagMapper.new`?

Reply:
I like `ProviderTagMapper`, we should also be looking at moving this into core's persister class since this kind of thing is [duplicated](https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/master/app/models/manageiq/providers/kubernetes/inventory/persister/container_manager.rb#L9) in a [few](https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/inventory/persister.rb#L15-L18) [places](https://github.com/ManageIQ/manageiq-providers-azure/blob/master/app/models/manageiq/providers/azure/inventory/persister.rb#L27-L33)

Reply:
https://github.com/ManageIQ/manageiq/pull/20561 starts this off